### DISCUSSION TITLE
iOS 403 App attestation failed but Android is fine?

### DISCUSSION THREAD

**Discussion by lukertaylor** [firebase/flutterfire#9467] [Q&A]
I just cannot get App Check to work with iOS. My config is fine with Android. I have created a key and used it to configure DeviceCheck. I am testing on a physical device (iPhone 6s). Running in debug mode I get the following when my app tries to wtite to Firestore:

```
9.4.0 - [FirebaseFirestore][I-FST000001] AppCheck failed: 'The operation couldn’t be completed. The server responded with an error: 
 - URL: https://firebaseappcheck.googleapis.com/v1/projects/my_project/apps/1:407394906485:ios:5094cebb0136b1f8ac233c:exchangeDebugToken 
 - HTTP status code: 403 
 - Response body: {
  "error": {
    "code": 403,
    "message": "App attestation failed.",
    "status": "PERMISSION_DENIED"
  }
}

```
I feel like I've tried everything. Any suggestions for how to fix this?

**Comment by cedvdb** [comment]
Have you resolved your issue ?
